configure.ac

```diff
@@ -147,7 +147,11 @@ fi
 # HTOP_CHECK_SCRIPT(LIBNAME, FUNCTION, DEFINE, CONFIG_SCRIPT, ELSE_PART)
 m4_define([HTOP_CHECK_SCRIPT],
 [
-   htop_config_script=$([$4] --libs 2> /dev/null)
+   if test ! -z "$htop_[$1]_config_script"; then
```

Comment:
please make the name of this variable all-uppercase (`as in HTOP_[$1]_CONFIG_SCRIPT`). the htop_* versions were meant to be "local" variables for the macro. Adding a short comment here explaining this was motivated by cross-compiling scenarios would also be welcome, for future maintenance.